src/onboarding/components/CompletionStep.tsx

```diff
@@ -60,7 +63,8 @@ class CompletionStep extends PureComponent<Props> {
             <div className="wizard-step--scroll-content">
               <h3 className="wizard-step--title">You are ready to go!</h3>
               <h5 className="wizard-step--sub-title">
-                Your InfluxDB Cloud has 1 organization, 1 user, and 1 bucket.
+                Your {this.environment} has 1 organization, 1 user, and 1
```

Comment:
I'm a little confused by this since it's been a while since i've onboarded, but does a CLOUD user ever hit this?

Reply:
I'm not entirely certain but since we have an environment variable and it's a simple if statement it seemed easiest to just handle the potential edge case.  If we know that this never gets hit by Cloud I'm happy to hard code the text here.

Reply:
Just to have this here in case anybody else finds it later: these pages are OSS only.